apps/web/playwright/fixtures/users.ts

```diff
@@ -6,9 +6,10 @@ import { hashSync as hash } from "bcryptjs";
 import dayjs from "@calcom/dayjs";
 import { DEFAULT_SCHEDULE, getAvailabilityFromSchedule } from "@calcom/lib/availability";
 import { prisma } from "@calcom/prisma";
+import { SchedulingType } from "@calcom/prisma/enums";
 import { MembershipRole } from "@calcom/prisma/enums";
 
-import { selectFirstAvailableTimeSlotNextMonth } from "../lib/testUtils";
+import { selectFirstAvailableTimeSlotNextMonth, teamEventSlug, teamEventTitle } from "../lib/testUtils";
```

Comment:
We should get teamEventTitle from the fixture itself now as it might be different from the default now.